### PR TITLE
Add 2021 sponsors

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

File: content/2021-online/index.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ Get an impression of PromCon EU 2019:
 
 A big thank you to all of our sponsors who are helping make PromCon happen! Click [HERE](http://cncf.io/sponsor) to learn more about how your company or organization can participate in the event as an event sponsor.
 
+<h3>Platinum</h3>
+<div class="sponsor-logos">
+  <a href="https://grafana.com/"><img src="/assets/grafana-2021.svg" class="logo"/></a>
```
Comment (roidelapluie) [inline content/2021-online/index.md:21]: can we have alt="" for accessibility?

File: content/2021-online/index.md [hunk 2]
```diff
@@ -16,6 +16,19 @@ Get an impression of PromCon EU 2019:
...
+</div>
+
+<hr>
+
+<h3>Gold</h3>
+<div class="sponsor-logos">
+  <a href="https://chronosphere.io/"><img src="/assets/chronosphere-2021.svg" class="logo"/></a>
```
Comment (roidelapluie) [inline content/2021-online/index.md:28]: can we have alt="" for accessibility?

File: content/2021-online/index.md (hunk 1 above)
Comment (RichiH) [reply content/2021-online/index.md]: done

File: content/2021-online/index.md (hunk 2 above)
Comment (RichiH) [reply content/2021-online/index.md]: done

**Review by roidelapluie — APPROVED** [review]
Thanks!

**Pull request merged**